### DISCUSSION TITLE
Found this new warning today. What is this about? K32 not valid any more?

### DISCUSSION THREAD

**Discussion by sirdmark** [Chia-Network/chia-blockchain#6166] [General]
Hello everyone,
after days of painstaking and eye damaging error correction in our ubuntu harvester setup, we found this warning today in our log file.

WARNING  Creating plots with size k=32, which is less than the minimum required for mainnet

Can someone explain please.

Thank you

**Comment by tiny0little** [comment]
for that sake of analysis, could you post version number you run, how it was installed (from .deb or github) and options for command ```chia plots create``` you use? also if you found this warning in debug.log or in plotter log

**Reply by sirdmark** [reply]
hello, 
ubuntu 1.1.6, github, chia plots create -k 32 -x -b 23900 -r 5 -u 64 -t ~/chia/temp -d ~/chia/final  , found the warning when plot was started. 
Thanks

**Reply by sirdmark** [reply]
Sorry, I know this was rude. There was someone at the door and I had to answer. We started the plot and the error message popped up. I can't read the log files at the moment because we have disconnected the kvm because we are upgrading memory in other machines.

**Reply by tiny0little** [reply]
`chia-blockchain/chia/plotting/create_plots.py`
```
if args.size < config["min_mainnet_k_size"] and test_private_keys is None:
    log.warning(f"Creating plots with size k={args.size}, which is less than the minimum required for mainnet")
```

could you post output of ```cat ~/.chia/mainnet/config/config.yaml | grep min_mainnet_k_size``` ?

**Reply by sirdmark** [reply]
Hello,
See above. Will take a few hours. About five pm GMT.

**Reply by sirdmark** [reply]
Hello,
sorry I can't read the logfiles atm. We have disconnected the kvm due to memory upgrade in other machines. It will take aprox. until 5pm GMT.

So I guess this is not related to a new minimum requirement being imposed?

Thanks

**Reply by sirdmark** [reply]
hello,
just connected. here is the output.

le@minecraft-server-1:~/chia-blockchain$ cat ~/.chia/mainnet/config/config.yaml | grep min_mainnet_k_size                                                                   2021-05-28 11:33:58.257 [INFO] min_mainnet_k_size: 64     

Thanks

**Reply by tiny0little** [reply]
here's your answer

**Reply by sirdmark** [reply]
Hello,
I wonder how that got in there. Must have been in the pack when we updated to 1.1.6. Highly unlikely we changed it. So I just change it to 32.

Thanks

**Reply by tiny0little** [reply]
> Must have been in the pack when we updated to 1.1.6

less likely

[initial-config.yaml](https://github.com/Chia-Network/chia-blockchain/blob/203232e429ca06cd6ec0fb23921b02c675d43dd5/chia/util/initial-config.yaml)
